include/filesystem_files_def.h

```diff
@@ -18,7 +18,7 @@ typedef enum NarcId
     NARC_a_0_1_0,
     NARC_a_0_1_1,
     NARC_a_0_1_2,
-    NARC_a_0_1_3,
+    NARC_msgdata_scenario_scr_msg,
```

Comment:
> msgdata/scenario/scr_msg.narc
A file already exists by this name, but the sha1 matches Diamond's version. I guess GF included it for some unknown reason?